docs_app/content/tutorial/7-flattening-operations.md

```diff
@@ -0,0 +1,132 @@
+# 7. Flattening Operations
+
+## TLDR:
+
+- "Flattening" is unwrapping nested (inner) Observables. (`Observable<Observable<T>> -> Observable<T>`)
+- There are four major flattening operations in RxJS: 
+    - **concat**: [`concatMap`](API), [`concatAll`](API), [`concat`](API), and [`concatWith`](API). Flattening one inner observable at a time, in order, without skipping any.
+    - **merge**: [`mergeMap`](API), [`mergeAll`](API), [`merge`](API), and [`mergeWith`](API). Flattening every inner observable, as soon as it can, without skipping any.
+    - **switch**: [`switchMap`](API), [`switchAll`](API). Flattening one inner observable at a time, but unsubscribing from it and "switching" to the newest one as soon as it arrives.
+    - **exhaust**: [`exhaustMap`](API), [`exhaust`](API). Flattening one inner observable at a time, skipping all other inner observables until the one it is subscribed to is completely exhausted (completed).
+- If you're dealing with HTTP, you *probably* wanted [`concatMap`](API) and not [`switchMap`](API).
+
+## Overview
+
+Since observables are containers that can contain themselves, they can be "unwrapped" or "flattened". This is a concept we touched on earlier in ["what is an operator?"](4-what-is-an-operator.md#Containers). The idea is simple enough, and can be demonstrated visually with an array: `[[1], [2, 3, 4], [5], [6, 7]]`, flattened, would be `[1, 2, 3, 4, 5, 6, 7]`. What we did was "unwrap" all of the inner arrays, and "flatten" their values into a single top level result. We can do the same thing with Observables, only to much greater effect.
+
```

Comment:
Would it be useful to also mention that these are sometimes called "higher-order" operators? Or do you prefer that we stop using that phrase?